Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM continuumio/miniconda3
+
+# if you forked pandas, you can pass in your own GitHub username to use your fork
+# i.e. gh_username=myname
+ARG gh_username=pandas-dev
+ARG pandas_home="/home/pandas"
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Configure apt and install packages
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends apt-utils dialog 2>&1 \
+    #
+    # Verify git, process tools, lsb-release (common in install instructions for CLIs) installed
+    && apt-get -y install git iproute2 procps iproute2 lsb-release \
+    #
+    # Install C compilers (gcc not enough, so just went with build-essential which admittedly might be overkill),
+    # needed to build pandas C extensions
+    && apt-get -y install build-essential \
+    #
+    # cleanup
+    && apt-get autoremove -y \
+    && apt-get clean -y \
+    && rm -rf /var/lib/apt/lists/*
+
+# Switch back to dialog for any ad-hoc use of apt-get
+ENV DEBIAN_FRONTEND=dialog
+
+# Clone pandas repo
+RUN mkdir "$pandas_home" \
+    && git clone "https://github.com/$gh_username/pandas.git" "$pandas_home" \
+    && cd "$pandas_home" \
+    && git remote add upstream "https://github.com/$gh_username/pandas.git"
```

Comment:
```suggestion
    && git remote add upstream "https://github.com/pandas-dev/pandas.git"
```

Reply:
I added that there in case a user wants to work directly with their own fork instead of the pandas-dev source. But I didn't document that well, guess I need to.

Reply:
I'm not sure I follow your logic here.

`origin` will already point to `$gh_username/pandas.git`.  Upstream, as we have in our documentation points to the original source.

Reply:
Oh you are right, my bad, I mixed the 2 steps up. To work with your own fork, you indeed only need to change where you clone from (origin), not the upstream.